autoscheduler/frontend/src/components/LandingPage/About/About.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import {
+  Link, Dialog, Typography, DialogContent, DialogTitle,
+} from '@material-ui/core';
+import IconButton from '@material-ui/core/IconButton';
+import CloseIcon from '@material-ui/icons/Close';
+import { withStyles } from '@material-ui/core/styles';
+
+import * as about from './About.css';
+
+const About: React.FC = () => {
+  const [open, setOpen] = React.useState(false);
+
+  const handleClickOpen = (): void => {
+    setOpen(true);
+  };
+  const handleClose = (): void => {
+    setOpen(false);
+  };
+
+  const WhiteTextTypography = withStyles({
+    root: {
+      color: '#FFFFFF',
+    },
+  })(Typography);
+
+  return (
+    <div className={about.container}>
+      <Link
```

Comment:
Looks like you're goimg to have to convert this to a flat `Button` and just change the styling

Reply:
I'd also be fine with just disabling this linter rule for this file, since the final HTML actually does render a button

Reply:
> his linter rule for this file, s

What do you mean by this?

Reply:
> I'd also be fine with just disabling this linter rule for this file, since the final HTML actually does render a button

What do you mean by this?